codegen/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojo.java

```diff
@@ -111,8 +114,11 @@ public void setIosCustomElements(String[] elements) {
     public void setAndroidCustomElements(String[] elements) {
         androidCustomElements = Arrays.asList(elements);
     }
-    
-    
+
```

Comment:
@mach6 I solved all issues in [codacy](https://www.codacy.com/app/SeLion/paypal-SeLion/pullRequest?prid=212622), but it still fails. Am i missing something?